Comparing Combat2.0 chunk2 alexL changes for the superbranch

1sourcecontrol opened the pull request:
I'm putting up this pull request so I can get more visibility and feedback. _These changes are not finished yet_. I'm, at least, missing explanations for the secondary skills.

Added suppression, stealth combat bonuses, d10 accuracy equations, etc.
Please give me feedback.

1sourcecontrol committed:
Update Basic Rules.txt

First step of adding suppression to the rules, tried to improve some text stuff

1sourcecontrol committed:
Update Basic Rules.txt

added in suppression fully, redid damage type and armor type

1sourcecontrol committed:
Update Basic Rules.txt

added to suppression definition

1sourcecontrol committed:
Update Basic Rules.txt

Added rules for stealth attacks, buffed sprinting and removed what appears to be contradicting statements, explicitly buffed weapon skill use while reloading, added 'hitting the dirt' effect, added shoving/pushing effects, and added minimums to miss chance.

1sourcecontrol committed:
Update Basic Rules.txt

beginning the incorporation of secondary skills, and alternative equations for combat accuracy (pertains primarily to DEX & PER)

1sourcecontrol commented:
Please read commit notes for finer details of what I changed. I tried to record things.

bleehu commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -131,15 +131,27 @@ Feats		- Your character will gain special perks as you level up. This
+stat, a secondary stat point is awarded (if the primary and secondary stats are
+associated). The maximum number of points that can be put into a given secondary stat
+is determined by the quantity of points in the largest associated primary stat.
+These stats are as follows (secondary stat associations are indented):
```
The Maximum number of stat points that can be given to a 2ndary stat point has to be equal to the max of the primary, right? If you only get them every time your base stat goes up?

1sourcecontrol replied:
> " The maximum number of points that can be put into a given secondary stat is determined by the quantity of points in the largest associated primary stat."

Means that if I have 8 points in STR and 10 points in DEX, I can put up to 10 points into movement and up to 8 points into Carry Ability. How can i reword this to be more clear?

Also, should I leave "Carry Ability" as it currently is spelled? All carry ability does is negate max-movement penalties from carrying heavy armor or large weapons. I just want it to be a term that is as elegant as possible.

bleehu replied:
Start a sentence with "If a secondary stat is contributed to by more than one primary stat..." because the literal wording you have indicates the opposite of what your comment says.

bleehu replied:
Having hit the Thesaurus, I can't find a more elegant term.

1sourcecontrol replied:
I guess we could try and call it, "carry strength", "endurance", "carry capacity", or "carry weight"
But I think "Carry Ability" is the most removed from common game terms. And I didn't want people to jump to the conclusion that this is some kind of universal every-object-has-weight system. Maybe that's unavoidable tho

trowl223 replied:
I don't see why we're moving to this system. Why not just add the "Carry Strength" stat and leave it at that?

1sourcecontrol replied:
The increased player choice, mainly.
1. If carry ability is simply added as a stat, it's worthless compared to getting pure dex for movement speed. Dex gets so much more, up until you hit the point cap. Then all that carry ability offers is cancellation of movements penalties so if you have no movement penalties, it's utterly worthless. I reckoned that having a different way to get the stat seemed prudent. 
2. Instead of forcing a player to pick 10 dex to increase movement, he can pick 10 dex or 10 str. Adding carry ability also gives STR, and especially fortitude, more value. The way it's previously been in CX, if you want to be good at melee you almost always have to get really good dex, because of movement speed. Then if you want a power fist to do damage, you need to have strength, so there is a bunch of your points decided for you by the game. Having a secondary stat system allows players to have more creativity and still be viable. 
3. Charisma has low value besides the extra feat bonus (which is arguably the only reason to take charisma because you can supplement charisma with speech skill, etc. but that's a discussion for another time). Skill points are such a fundamental part of the game, you _have_ to put lots of points into intelligence if you want to be good at RP or reloading your weapon, etc. The secondary stat system allows you to pick charisma or intelligence to increase your skill point gain.

As usual, i'm open to ideas and I'm trying to get the most elegant and useful system built, that I can.

bleehu commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -167,6 +179,23 @@ and the score of your roll would be
 That's a really high roll! But that may or may not be enough to lift the object
 depending on how heavy it is!
 
+Ranged or melee weapon accuracy bonuses use slightly different equations, as combat
```
different from what? Each other?

bleehu commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -167,6 +179,23 @@ and the score of your roll would be
+accuracy). Your total bonus accuracy before rounding is:
+0.5 + 0.6 = 1.1
+Which you would round to 2 bonus accuracy (-2 miss chance).
+
```
Please separate each example into separate paragraphs

bleehu replied:
looks good

bleehu commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -441,40 +472,53 @@ to subdue your opponent (either through choke-out, pin, or a joint lock). You
+firing your weapon after doing so will cause you to incur a +1 miss chance
+penalty. The penalty occurs during the actions you sprint, as well as 2 actions 
+afterwards. You can stack sprinting a number of Actions equal to your Fortitude.
 You can jump horizontally Dex/2 meters, and add 1 meter if you have a 5 meter 
```
Please reword 'stack sprinting actions' to 'sprint for a number of subsequent actions equal to' for the sake of clarity

bleehu commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -521,6 +565,51 @@ Hunkering costs one action, and you cannot fire your weapons. Imagine
+action, you incur bonuses depending on the circumstances of the attack.
+	Ranged weapon stealth-attacks (including explosive but not any other thrown 
+weapons) have normal damage with -1 miss chance, if 'adequate' aiming time is 
+given (DM's discretion: how rushed you are, or how skilled you are at stealth, etc).
```
Why remove the effects of extra lethality of a stealth ranged kill?

1sourcecontrol replied:
To my knowledge, there isn't any extra lethality to a ranged stealth kill? Do we normally apply some kind of bonus?
In one sense I improved the lethality by making it so you can land hits more easily.

bleehu replied:
It used to be an auto-crit as well as an accuracy bonus.

trowl223 replied:
Yeah, there should be a mention of crit here. Also I'm not sure oin the "Hit the Dirt" thing. Maybe after a Fort DC?

1sourcecontrol replied:
You're viewing an older commit, the crit effect is already in.

The "Hit the Dirt" isn't supposed to be a choice because it represents natural instincts of self-preservation and/or any sort of military experience. Unless the fantasy world gives the player the choice to eliminate fear and primal instincts of their character, they'd probably dive to the ground when they get shot at. Unlike how the combat is played right now, where every character could stand around in a circle and take turns ghatting each other if everyone so chose. Also I'd, personally, prefer to remove it from the superbranch altogether than slow down turns more with a check.

Also, if a  player gets caught out in the open and the opponent has 3 actions of 6-bullet automatic fire to spray on them, this will help prevent the game from being overly punishing of single mistakes or single low roll.  And vice-versa, the enemies won't be walked over as easily, but I don't think that is necessarily a good thing.

And this all ties into suppression as a concept: has more to do with realism, self-preservation, and fear of death than anything else. But I'm trying to make it game-playable too.

bleehu commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -521,6 +565,51 @@ Hunkering costs one action, and you cannot fire your weapons. Imagine
+suppression AOE (Area-of-Effect, typically a 1-3m circle).
+If a suppressed target moves out into the open**, they trigger automatic attack
+rolls (or automatic hits) by the suppressor, once per target per suppressor's 
+turn. If the suppressed target shoots, they fire with +1 miss chance. Suppression
```
Does that +1 stack per number of enemies suppressing?

bleehu commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -521,6 +565,51 @@ Hunkering costs one action, and you cannot fire your weapons. Imagine
+"covering fire", or "suppressive fire" is the art of shooting a hail of 
+bullets (as accurately as possible) to try and prevent the enemy from 
+effectively fighting back while your team maneuvers. Suppression comes in
+three flavors: light, medium, and heavy; these indicate how wide of an area
```
What do the three flavors do?

1sourcecontrol replied:
I was going to put the light-medium-heavy suppression definitions in the weapons manual, but I guess it makes sense to define them here

bleehu replied:
I could see good arguments for either spot, but I think I would expect to look for them here.

bleehu commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -570,29 +660,42 @@ unconsciousness. DC 50, Fortitude. Greater / Lesser poison increases damage
+	Ballistic weapons are straightforward: they are blocked by armor and ignore shields.
+	Any damage that defeats armor damages health.
+Plasma Weapons:
+	Plasma interacts fairly neutrally with both armor and shields. Armor's effectiveness
```
Can we get something more specific than 'fairly neutrally'? What does that even mean?

1sourcecontrol replied:
That's a relic from the past  text that I reworded. I'll remove/reword

bleehu replied:
Thankya!

bleehu commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -570,29 +660,42 @@ unconsciousness. DC 50, Fortitude. Greater / Lesser poison increases damage
+Laser Weapons:
+	Lasers are completely blocked by shields, dealing all of their damage to the shielding.
+	Lasers deal full damage to armored targets; If a player wearing ballistic armor is hit 
+	with a laser, they receive damage as if they weren't wearing any armor.
```
that is very clearly written! Very good! I like the semi-rock-paper-scissors created by laser-balistic-plasma

1sourcecontrol replied:
Just for the record, I'm just trying to define the way we are playing now. Not make anything new (besides the changes to plasma DOT).

1sourcecontrol replied:
And thanks! :)

bleehu commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -570,29 +660,42 @@ unconsciousness. DC 50, Fortitude. Greater / Lesser poison increases damage
+Explosives:
+	Certain grenades and devices have Explosive (concussive) damage. Explosive Damage ignores
+	armor (explosive damage is essentially, a crushing force). Fragmentation grenades are 
+	considered ballistic weapons, as their primary damage delivery method is shrapnel. 
```
How do they interact with Shields? Do they ignore shields as well?

1sourcecontrol replied:
Should they ignore shields as well? I think we have been treating explosives as ignoring both CS shields and armor. Unless you had a paladin plasma shield or something.

bleehu replied:
I'm a little hesitant to make a damage type that ignores both armor and shields entirely, but I think it makes sense. I just worry that it's dangerous to implement weapons that use it with comparable damage to others down the line.

1sourcecontrol replied:
I'm hoping to give level 10ish players purchase access to better armors that target specific damage types. Such as EOD armor for explosives, ceramic/ablative/reflective armors for defending against lasers, etc. It would increase complexity, but players should have a very good understanding of gameplay by level 10, and it gives us a lot of creative options to add to the otherwise boring vanilla armor.

bleehu reviewed (changes requested):
Altogether pretty good. Some things to clarify, then let's shipit!

1sourcecontrol committed:
Update Basic Rules.txt

per bleehu's comment, updated secondary stat point description for clarity. Also added some secondary stat point examples.

1sourcecontrol committed:
Update Basic Rules.txt

buffed horizontal jumping, buffed stealth ranged damage... made text changes and added in the definitions of light/med/heavy suppression

1sourcecontrol committed:
Merge branch 'combat2.0superbranch_AlexL' into combat2.0_chunk2_AlexL

1sourcecontrol committed:
Update Basic Rules.txt

1sourcecontrol commented:
@bleehu, any other thoughts before I merge?

bleehu commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -194,8 +194,8 @@ and the score of your roll would be
-accuracy rolls are played out on a d10 scale:
+Ranged and melee weapon accuracy bonuses use a slightly different equation than the 
+above Stat Bonus equation, as combat accuracy rolls are played out on a d10 scale:
 
```
Much clearer! Good work!

trowl223 replied:
Why the change?

bleehu replied:
Alex is hoping to reduce turn time by reducing calculation complexity.

1sourcecontrol replied:
I guess @trowl223 you could be referring to a couple of changes, I put the equation in because of the d10 accuracy change (like Michael referenced). The equation is different in more than just scale because I wanted to make base stats less powerful in comparison to attachments when you min-max.

bleehu commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -648,29 +650,45 @@ action of their turn. Suppression, also known as, "laying down fire",
+	automatically hit x2 and roll for armor. You suppress all targets within a 2m radius 
+	AOE around the primary target.
+
 **"Moving out into the open" includes moving into a new location while you can
```
Great addition! I'm looking forward to playing this!

bleehu reviewed (approved):
shipit!

trowl223 commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -581,13 +604,14 @@ to roll either for (15 - FORT) turns, or until they pass.
 Bleeding - d10/2+1 (drop the remainder) stackable damage per turn. Broken with 
 Medicine Skill check. DC varies. If you have a bleed condition, seek medical aid
-immediately, You CAN Bleed Out, you have been warned.
+immediately, You can Bleed Out.
```
Why the removal here?

1sourcecontrol replied:
I removed it because it seemed a tad too unprofessional and it's restating the obvious.

trowl223 replied:
I kinda like the lightheartedness of the rules making little jokes. It gives the game a little character instead of being a cold and clinical presentation of rules.

1sourcecontrol replied:
I understand, I like jokes in the rules, too. Jokes can be difficult though, the reader has to interpret the meaning without any cues.
I didn't take this as a joke at all, it came across as unnecessary emphasis.

trowl223 commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -581,13 +604,14 @@ to roll either for (15 - FORT) turns, or until they pass.
-die. While in this state, you cannot take any actions, but can mumble or speak 
+your Fort score, for your lifespan. when you have failed more times than 
+your Fort score, you die. While in this state, you cannot take any actions,
+but can mumble or speak 
```
What is a "lifespan?"

1sourcecontrol replied:
the life of the character, i'll change the text

1sourcecontrol replied:
although i made the boo-boo of thinking that we weren't playing according to the rules with #133

trowl223 commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -250,17 +279,17 @@ Species. See chapter 1: Races if you haven't yet.
 Leveling up:
 Gain feats according to the table below.
-With Charisma 9 or higher, receive more feats. The feats listed in the table are totals, 
+With Intelligence 9 or higher, receive more feats. The feats listed in the table are totals, 
```
Why the switch here?

1sourcecontrol replied:
Part of the re-balance; something I haven't added yet: charisma granting a unique feat if you put enough points into it.

1sourcecontrol replied:
Because of secondary skills putting allowing skill point gain to happen with Charisma and Intelligence, intelligence can easily become a dump stat if you don't need nanites. Moving the charisma feat bonus to intelligence and giving charisma a unique feature, seemed like an appropriate balance.

trowl223 commented:
Sorry to be late to the discussion. Awareness and Perception checks are still 1 action to perform, as they entail using your skill to pause and /inspect/ your environment. General awareness (who is where, what are targets I can shoot at, what sort of cover can I move to) is a free action. That is why there is a distinction between skill checks and "Getting Your Bearings / Non Skill-Based Perception."

I've also left comments in places where I was confused on an edit. Anywhere I didn't leave a comment, I really want to compliment @1sourcecontrol for his thoroughness, thoughtfulness and overall keen writing ability! He is doing a tremendous job at moving us into a combat system which is much more robust and dynamic!

1sourcecontrol commented:
@trowl223  Ah, okay. I assumed we were doing the typical 1 or 2 free skill rolls per combat turn, depending on the circumstances, my mistake.

Thanks so much!

1sourcecontrol committed:
Update Basic Rules.txt

Added "lucky" feat

1sourcecontrol committed:
Update Basic Rules.txt

added text alluding to the new charisma feats, going to put the new feats into the feat.txt. Rather than the basic rules

1sourcecontrol committed:
Update 03_Feats.txt

added lucky, positional mastermind, and will to survive feats. Various spelling fixes

1sourcecontrol committed:
Update Basic Rules.txt

tried to make awareness reference more clear, fixed inconsistency with positional mastermind feat

1sourcecontrol committed:
Update 03_Feats.txt

changed bonus feats to allow them to be gained later in the game, if the character's stats increase

1sourcecontrol committed:
Update Basic Rules.txt

changed some rules for adding up combat accuracies, and I tried to expand the description to include a tutorial piece.

bleehu commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -527,6 +527,73 @@ Prerequisites: Level 10, Twitch
+	Good things predictably happen to you.
+	Luck = 6	
+		You can choose to reroll a die, once per long rest. You can only reroll 
+		once per separate die use and if you reroll, you must keep the second roll.
```
Do we have a formal definition of 'short rest' or 'long rest'?

1sourcecontrol replied:
I don't know... I thought Taylor made reference to it. I'll add it to the basic rules

bleehu commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -527,6 +527,73 @@ Prerequisites: Level 10, Twitch
+to their environments, allowing you to feint and position yourself in cover better. 
+If you have 9 Charisma, any living creature targeting you gets a +1 miss chance penalty
+when you have any kind of cover. If you have 12 Charisma, anything targeting you gets a 
++1 miss chance penalty when you have any kind of cover.
```
Feats need to be a separate feats document. Not in basic Rules.

bleehu replied:
The feats themselves are good though.

1sourcecontrol replied:
These feats are in the feats doc; please check the header.

bleehu replied:
whoops! dorp.

1sourcecontrol committed:
partial buff to prone, added flanking

increased gun miss chance bonus while prone -> a partial buff: by giving it a whole number, players won't have to round on the fly. If their numbers are already rounded, the ending miss chance will be the same 5.5+0.5 =6 ->  5+1 =6. However, if the player didn't have to round in earlier calculations, then they would gain an accuracy point: 5 + 0.5 = 5 (after rounding) -> 5+1 = 6.

I think it's worth the buff. I'm trying to reduce the amount of head calcs the player has to do on the fly, specifically, i'm trying to prevent players from needing to round.

Also added flanking.

1sourcecontrol committed:
updated blindfiring and other text

added the ability to blind fire while hunkered, like the old feat does. tried to improve the blind fire description, and stat bonus roll description

1sourcecontrol committed:
changed blind fire feat

since blind fire is implemented for all players, changed the blindfire feat to make it relevant

1sourcecontrol committed:
Improving hip fire text

1sourcecontrol committed:
spellcheck, and definition improvement

Trying to improve the definition of partial and full cover. If you're behind full cover (up to your neck) you shouldn't be able to crouch and get a bonus. You'll just be completely obscured (wasn't sure how to add that definition, if I needed to). 
If you're behind full cover, you shouldn't normally be able to go prone and gain +6 cover bonus.

1sourcecontrol committed:
crit hit alteration

changed the crit system to drop the remainder (rounddown towards 0) to rebalance how the crit range is calculated. 15th atrib point is much more important this way

1sourcecontrol committed:
put in a descr for shields, updated cloaking

updated cloaking and shields to C2.0

1sourcecontrol committed:
improving sentence structure of shields

1sourcecontrol committed:
armor and shields outline

1sourcecontrol committed:
added two levels of armor material

also made the framework for future armor

1sourcecontrol committed:
Adding a common-sense case to suppression rules

1sourcecontrol committed:
text improvement

1sourcecontrol committed:
added all the standard armor levels

1sourcecontrol committed:
defining secondary stats and max movement speed penalties

1sourcecontrol committed:
simplifications

Simplified how secondary stats work, tried to make some text indicate how much movement per action instead of movement per two actions, and I tried to make 'max movement speed' more simple by turning them into just 'movement speed penalties'

1sourcecontrol committed:
clarified movement speed

specified that you should round movement speed penalties

1sourcecontrol committed:
added armor rig, ammo storage, & equipment introductions

1sourcecontrol committed:
removed an extra linespace

1sourcecontrol committed:
added referral to the basic rules

1sourcecontrol commented on play/PlayerRules/CharacterCreation/05_ArmourList.txt:
```diff
@@ -19,16 +19,157 @@
+make it easy to attach equipment. When you purchase equipment, it will list how
+many Hard Points it consumes. You can't attach more equipment than you have available
+Hard Point space.
+
```
@bleehu, how do these rules look?

1sourcecontrol committed:
added resting rules; minor other changes

also reverted some flavor text. 
also passed the down system forward until the down system issue is resolved
